assets/javascripts/kitten/components/cards/project-card.js

```diff
@@ -120,17 +123,57 @@ class ProjectCardComponent extends React.Component {
   }
 
   renderTags() {
-    if (!this.props.tags) return
-
     return (
-      <div className="k-ProjectCard__grid">
+      <div key={ `tag-list-${Math.random(1)}` } className="k-ProjectCard__grid">
         <Marger top="1.3" bottom="1.3">
           <TagList items={ this.props.tags } tiny />
         </Marger>
       </div>
     )
   }
 
+  renderTagsInList(tags, index) {
+    const renderSeparator =
+      <div className="k-u-margin-left-single">
+        <hr className="k-ProjectCard__tagLists__separator k-Separator--darker"/>
+      </div>
+
+    const renderTagListWithMargin =
+      <div
+        key={ `tag-with-separator-${Math.random(1)}`}
+        className="k-u-margin-left-single">
+        <TagList tags={ tags } tiny />
+      </div>
+
+    const renderBlock = (index != 0)
+      ? [renderSeparator, renderTagListWithMargin]
+      : <TagList tags={ tags } tiny />
+
+    return (
+      <div
+        key={ `tag-list-block-${Math.random(1)}` }
+        className="k-ProjectCard__grid--flex">
+        { renderBlock }
+      </div>
+    )
+  }
+
+  renderTagLists() {
+    return (
+      <div className="k-ProjectCard__grid">
+        <Marger top="1.3" bottom="1.3" className="k-ProjectCard__grid--flex">
+          { this.props.tagLists.map(this.renderTagsInList) }
+        </Marger>
+      </div>
+    )
+  }
+
+  renderTagsArea() {
+    if (!this.props.tagLists && !this.props.tags) return
+
+    return this.props.tagLists ? this.renderTagLists() : this.renderTags()
```

Comment:
On pourrait s'éviter la méthode `renderTags` en mettant la condition dans `renderTagLists` avec quelque chose comme : 
```js
const tags = this.props.tagLists || this.props.tags
…
{ tags.map(this.renderTagsInList) }
```

Reply:
Hum c'est à cause des props `items` et `tags` qui font la même chose mais qui ne se nomment pas pareil suivant la version...

Reply:
Ah oui, mais si on met `this.props.tags` dans un objet, on devrait se retrouver avec la même structure ? 

```js
const tags = this.props.tagLists || { items: this.props.tags }
…
{ tags.map(this.renderTagsInList) }
```